src/Native/Runtime/GCMemoryHelpers.inl

```diff
@@ -7,14 +7,19 @@
 // Unmanaged GC memory helpers
 //
 
+#ifndef DACCESS_COMPILE
 #ifdef WRITE_BARRIER_CHECK
 extern uint8_t* g_GCShadow;
 extern uint8_t* g_GCShadowEnd;
-extern "C" uint8_t* g_lowest_address;
-extern "C" uint8_t* g_highest_address;
+typedef DPTR(uint8_t)   PTR_uint8_t;
+GPTR_DECL(uint8_t, g_lowest_address);
```

Comment:
`g_lowest_address/g_highest_address` are `extern "C"` in other places. Make them `extern "C"` here as well?